docs/en/get_started.md

```diff
@@ -1,213 +1,98 @@
 
 ```shell
 git clone https://github.com/open-mmlab/mmsegmentation.git
 cd mmsegmentation
-pip install -e .  # or "python setup.py develop"
+pip install -v -e .
+# "-v" means verbose, or more output
+# "-e" means installing a project in editable mode,
+# thus any local modifications made to the code will take effect without reinstallation.
 ```
 
-:::{note}
-
-1. When training or testing models on Windows, please ensure that all the '\\' in paths are replaced with '/'. Add .replace('\\', '/') to your python code wherever path strings occur.
-2. The `version+git_hash` will also be saved in trained models meta, e.g. 0.5.0+c415a2e.
-3. When MMsegmentation is installed on `dev` mode, any local modifications made to the code will take effect without the need to reinstall it.
-4. If you would like to use `opencv-python-headless` instead of `opencv-python`,
-   you can install it before installing MMCV.
-5. Some dependencies are optional. Simply running `pip install -e .` will only install the minimum runtime requirements.
-   To use optional dependencies like `cityscapessripts`  either install them manually with `pip install -r requirements/optional.txt` or specify desired extras when calling `pip` (e.g. `pip install -e .[optional]`). Valid keys for the extras field are: `all`, `tests`, `build`, and `optional`.
-   :::
-
-### A from-scratch setup script
-
-#### Linux
-
-Here is a full script for setting up mmsegmentation with conda and link the dataset path (supposing that your dataset path is $DATA_ROOT).
+Case b: If you use mmsegmentation as a dependency or third-party package, install it with pip:
 
 ```shell
-conda create -n open-mmlab python=3.10 -y
-conda activate open-mmlab
-
-conda install pytorch=1.11.0 torchvision cudatoolkit=11.3 -c pytorch
-pip install mmcv-full -f https://download.openmmlab.com/mmcv/dist/cu113/torch1.11.0/index.html
-git clone https://github.com/open-mmlab/mmsegmentation.git
-cd mmsegmentation
-pip install -e .  # or "python setup.py develop"
-
-mkdir data
-ln -s $DATA_ROOT data
+pip install mmsegmentation
 ```
 
-#### Windows(Experimental)
-
-Here is a full script for setting up mmsegmentation with conda and link the dataset path (supposing that your dataset path is
-%DATA_ROOT%. Notice: It must be an absolute path).
+## Verify the installation
 
-```shell
-conda create -n open-mmlab python=3.10 -y
-conda activate open-mmlab
+To verify whether MMSegmentation is installed correctly, we provide some sample codes to run an inference demo.
 
-conda install pytorch=1.11.0 torchvision cudatoolkit=11.3 -c pytorch
-set PATH=full\path\to\your\cpp\compiler;%PATH%
-pip install mmcv
+**Step 1.** We need to download config and checkpoint files.
 
-git clone https://github.com/open-mmlab/mmsegmentation.git
-cd mmsegmentation
-pip install -e .  # or "python setup.py develop"
-
-mklink /D data %DATA_ROOT%
+```shell
+mim download mmsegmentation --config pspnet_r50-d8_512x1024_40k_cityscapes --dest .
```

Comment:
might use `wget`

Reply:
I create a new pr to solve the mim download error. #1633